### PR TITLE
Move Terms to Privacy

### PR DESCRIPTION
**Pull request by MatkoTom**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant
motivation and context. List any dependencies that are required for this change.

Moved Terms and Conditions from Help screen to Privacy screen.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can
reproduce. Please also list any relevant details for your test configuration. All testing was done
manually by following, plainly, functionality requirements. If there are some special edge cases or
special ways that things need to be tested they will be mentioned below.

Dev testing

**Test Configuration**:

* Firmware version: G980FXXSFFVHA
* Hardware: SAMSUNG Galaxy S20
* SDK: Android 13

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream module

### CODE REVIEW

File: app/src/main/java/com/clover/studio/spikamessenger/ui/main/settings/SettingsFragment.kt [hunk 1]
```diff
@@ -197,9 +197,9 @@ class SettingsFragment : BaseFragment() {
...
 
-        binding.flHelp.setOnClickListener {
-            goToHelp()
-        }
+//        binding.flHelp.setOnClickListener {
+//            goToHelp()
+//        }
 //
 //        binding.clChat.setOnClickListener {
 //            goToChatSettings()
```
Comment (ibencetic-CS) [inline app/src/main/java/com/clover/studio/spikamessenger/ui/main/settings/SettingsFragment.kt:205]: Can we just delete these instead of keeping them commented out?

File: app/src/main/res/layout/fragment_settings.xml [hunk 2]
```diff
@@ -233,29 +233,29 @@
...
-        app:layout_constraintTop_toBottomOf="@id/cl_privacy">
+    <!--    <FrameLayout-->
+    <!--        android:id="@+id/fl_help"-->
+    <!--        android:layout_width="match_parent"-->
+    <!--        android:layout_height="wrap_content"-->
+    <!--        android:layout_margin="@dimen/ten_dp_margin"-->
+    <!--        android:padding="@dimen/twenty_dp_margin"-->
+    <!--        app:layout_constraintEnd_toEndOf="parent"-->
+    <!--        app:layout_constraintStart_toStartOf="parent"-->
+    <!--        app:layout_constraintTop_toBottomOf="@id/cl_privacy">-->
```
Comment (ibencetic-CS) [inline app/src/main/res/layout/fragment_settings.xml:244]: Can also be deleted?

**Review by ibencetic-CS — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/clover/studio/spikamessenger/ui/main/settings/SettingsFragment.kt (hunk 1 above)
Comment (MatkoTom) [reply app/src/main/java/com/clover/studio/spikamessenger/ui/main/settings/SettingsFragment.kt:205]: This layout is part of the Figma design, might get implemented later. But, it shouldn't be too much of a hassle to add it back.

File: app/src/main/res/layout/fragment_settings.xml (hunk 2 above)
Comment (MatkoTom) [reply app/src/main/res/layout/fragment_settings.xml:244]: Check comment above

**Review by ibencetic-CS — APPROVED** [review]
Bit ce to dobro!
